Source/AliveLib/Movie.cpp

```diff
@@ -408,7 +408,10 @@ EXPORT char CC DDV_Play_Impl_4932E0(const char* pMovieName)
 
             if (sFrameInterleaveNum_5CA23C > 15)
             {
-                if (Input_IsVKPressed_4EDD40(VK_ESCAPE) || Input_IsVKPressed_4EDD40(VK_RETURN))
+                sInputObject_5BD4E0.Update_45F040();
+                if (Input_IsVKPressed_4EDD40(VK_ESCAPE) || Input_IsVKPressed_4EDD40(VK_RETURN)
+                    || sInputObject_5BD4E0.isPressed(InputCommands::eFartOrRoll) || sInputObject_5BD4E0.isPressed(InputCommands::ePause)
```

Comment:
This is just wrong. farting or rolling actions shouldn't be connected with the movie playback

Reply:
Hmm - what does PSX do? Given there isn't escape/enter keys